### PR TITLE
added modal and rating

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

**Review by gauravsinhaweb — APPROVED** [review]
Looks good to me!

File: src/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  backdrop-filter: blur(1px);
+}
+
+.modal {
+  border-radius: 8px;
+  border: 1px solid var(--BLACK);
+  background-color: var(--WHITE);
+  max-width: 350px;
+  padding: 10px;
+}
```
Comment (manojsarna) [inline src/components/modal/modal.css:19]: Suggestion : can you use rems instead of px values

File: src/components/modal/modal.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const modalShowBtn = document.getElementById("modal-btn");
+const modalCloseBtn = document.getElementById("modal-close-btn");
+const modal = document.getElementById("modal");
+
+modalShowBtn.addEventListener("click", () => {
+  modal.style.display = "flex";
+});
+
+modalCloseBtn.addEventListener("click", () => {
+  modal.style.display = "none";
```
Comment (manojsarna) [inline src/components/modal/modal.js:10]: Looks Good!!

File: src/components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  align-items: center;
+  backdrop-filter: blur(1px);
+}
+
+.modal {
+  border-radius: 8px;
+  border: 1px solid var(--BLACK);
+  background-color: var(--WHITE);
+  max-width: 350px;
+  padding: 10px;
```
Comment (PuneetP16) [inline src/components/modal/modal.css:18]: Why have you used pixels over here?

File: src/components/rating/rating.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.rating {
+  display: inline-flex;
+  gap: 6px;
+  padding: 0.5rem 1rem;
+  font-size: 1.4rem;
+}
+
+.rating i:hover {
+  color: rgb(255, 224, 46);
+  font-weight: 800;
```
Comment (PuneetP16) [inline src/components/rating/rating.css]: Suggestion: use variables instead of direct values.

**Review by manojsarna — APPROVED** [review]
Overall code looks good!!

Do Check the comments

**Review by PuneetP16 — APPROVED** [review]
Implementation looks neat, good work, go ahead and merge!

File: src/components/modal/modal.css (hunk 1 above)
Comment (pankaj-prog) [reply src/components/modal/modal.css:19]: i don't want the width of modal to be greater than the mobile screen width

File: src/components/modal/modal.css (hunk 3 above)
Comment (pankaj-prog) [reply src/components/modal/modal.css:18]: i don't want the width of modal to be greater than the mobile screen width

**Pull request merged**